web/src/components/core/Section.jsx

```diff
@@ -86,7 +88,11 @@ export default function Section({
     const headerText = !path?.trim() ? title : <Link to={path}>{title}</Link>;
 
     return (
-      <h2 id={headerId}>{headerIcon}<span>{headerText}</span></h2>
+      <header>
+        <h2 id={headerId}>{headerIcon}<span>{headerText}</span></h2>
+        <p>{description}</p>
```

Comment:
By now, I'm not adding an `aria-describedby` to the section pointing to this. I'm not sure if

* It's actually needed, since the description will be visible and I guess it will be read aloud
* It will be read twice
* It's needed to add an aria-hidden here too.

So, let's postpone this small detail just a bit, until we have time to test these a11y things.